docs/userGuide/tipsAndTricks.md

```diff
@@ -34,18 +34,33 @@ For Markdown syntax: To display a literal character that are normally used for M
 
 ##### :fas-lightbulb: Using {% raw %}{% endraw %} to display `{{ content }}`
 
+By default, MarkBind processes any code in the form of `{{ content }}`. This is because Nunjucks (which is [supported by MarkBind](https://markbind.org/userGuide/markBindSyntaxOverview.html#support-for-nunjucks)) uses this syntax to [evaluate an expression, variable, or function calls](https://mozilla.github.io/nunjucks/templating.html#variables). For instance:
```

Comment:
Let's fix this link to use relative link so that it does not link out of the PR preview to markbind.org

Reply:
Fixed in 5789e6e!